README.md

```diff
@@ -1175,7 +1186,22 @@ POST /policies
       "service-offerings": [
         "p-config-server"
       ]
-    }
+    },
+    {
+    "message-policies": [
+        {
+            "id": "b56fdcce40fd0dbe1c74791343850f6881595b26",
```

Comment:
Is this attribute `id` truly needed? Probably ok to remove.

Reply:
Removed